zerver/tests/test_auth_backends.py

```diff
@@ -1113,7 +1113,7 @@ def test_github_oauth2_email_no_reply_dot_github_dot_com(self) -> None:
         # As emails ending with `noreply.github.com` are excluded from
         # verified_emails, choosing it as an email should raise a `email
         # not associated` warning.
-        account_data_dict = dict(email="hamlet@noreply.github.com", name=self.name)
```

Comment:
This change should probably be combined with the previous commit instead of making a separate commit

Reply:
I squashed these and pushed back to the PR.